### DISCUSSION TITLE
”返回旧版“是不是失效了？

### DISCUSSION THREAD

**Discussion by EarOfWheat** [the1812/Bilibili-Evolved#3924] [综合讨论 / General]
点击“返回旧版”按钮没用。虽然这不是脚本的功能...(￣﹃￣)
![image](https://user-images.githubusercontent.com/74401501/211792699-45c37055-147b-494a-99f0-436390ff6f77.png)

**Comment by SunJunge08** [comment]
你可以试试清除cookies，重新登录，再按Ctrl+F5，应该就好了
